wordpress-mobile/WordPress-Android
# Jetpack focus show or hide automattic family section on about screen

This PR adds config for showing or hiding the following
- About Automattic Family row
- About Footer (Automattic logo)

Depends on corresponding [about-automattic-library PR](https://github.com/Automattic/about-automattic-android/pull/39)

| WordPress | Jetpack |
|---|---|
|![Screenshot_20221006_164935](https://user-images.githubusercontent.com/990349/194225354-7a5069e1-95d4-4808-82aa-d06b1671b568.png)|![Screenshot_20221006_170517](https://user-images.githubusercontent.com/990349/194226699-7cc18257-a122-49b3-a248-8a52d370b63d.png)|



To test:

Test 1:
- Build and Run WPAndroid 
- Go to Me > About WordPress
- Ensure About Automattic Family row is not shown 
- Ensure automattic logo is not shown at the bottom

Test 2:
- Build and Run JPAndroid 
- Go to Me > About Jetpack
- Ensure About Automattic Family row is shown
- Ensure About Automattic Family doesn't have WordPress bubble in it
- Ensure automattic logo is shown at the bottom

Test 3:
- Check WP twitter link goes to [@WPAndroid](https://twitter.com/wpandroid)
- Check JP twitter link goes to [@jetpack](https://twitter.com/jetpack)

Test 4:
- Check WP blog title is **News** and points to https://wordpress.org/news/ 
- Check JP blog title is Blog and points to https://jetpack.com/blog/

Test 5:
- Check WP Contribute is present and points to https://make.wordpress.org/mobile/handbook
- Check JP has Work With Us and points to https://automattic.com/work-with-us

NOTE:
- Terms of service and Privacy policy links will be updated later along with  links on Play store


## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested both WPAndroid and Jetpack variants

3. What automated tests I added (or what prevented me from doing so)
Existing unit tests



PR submission checklist:
